jwt.go

```diff
@@ -150,8 +150,11 @@ type JWTAuth struct {
 	logger        *zap.Logger
 	parsedSignKey interface{} // can be []byte, *rsa.PublicKey, *ecdsa.PublicKey, etc.
 
-	jwkCache     *jwk.Cache
-	jwkCachedSet jwk.Set
+	// JWK cache by resolved URL to support placeholders in JWKURL
+	jwkCaches map[string]*struct {
```

Comment:
Since the keys are populated during requests handling, I believe it should be guarded with `sync.Map` to allow concurrent access. No?